### PR TITLE
Add changelog for 0.5.0

### PR DESCRIPTION
**Pull request by kezhenxu94**

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -35,6 +35,4 @@ github:
         strict: true
         contexts:
           - CheckStatus
-      required_pull_request_reviews:
-        dismiss_stale_reviews: true
-        required_approving_review_count: 1
```
Comment (kezhenxu94) [inline .asf.yaml:40]: I'm removing this since trivial changes like this PR can be merged quickly as long as the checks passed

Comment (fgksgf) [reply .asf.yaml:40]: How about non-trivial changes ?

Comment (kezhenxu94) [reply .asf.yaml:40]: > How about non-trivial changes ?

Committers can request reviews from other committers, as we have limited committers in this repo, I'd rather work on trust that we know the changes are non-trivial and need review

Comment (fgksgf) [reply .asf.yaml:40]: Can those people who are not members (committer/PMC) merge directly when the checks passed ? If they can't I think this change of `.asf.yml` is ok.

Comment (kezhenxu94) [reply .asf.yaml:40]: > Can those people who are not members (committer/PMC) merge directly when the checks passed ? If they can't I think this change of .asf.yml is ok.

Of course they can't, merging needs write access to the repo

Comment (fgksgf) [reply .asf.yaml:40]: Ok, I have no worries.

**Review by fgksgf — APPROVED** [review]

**Pull request merged**